pipeline_games/requirements.txt

```diff
@@ -1,4 +1,7 @@
 bs4 
 pandas
+psycopg2-binary
 pylint
```

Comment:
Not needed import unless you're assuming anyone using this file will need to pylint it but not needed for Docker. Although similar could be said for pytest - since you're including testing files (not in Dockerfile though but in general) it is needed

Reply:
I guess we could make different requirements for docker? I keep these in there for the automated deployment bit that would need them but can take them out

Reply:
Automated deployment doesn't need it in requirements (I think)

Reply:
It looks better as you dont have to write out each library when you want to run pytest but will deal with that problem when and if we get there